default/scripting/species/common/industry.macros

```diff
@@ -5,15 +5,25 @@ BASIC_INDUSTRY
 '''        EffectsGroup
             scope = Source
             activation = And [
-                Source
                 Planet
                 TargetIndustry low = 0
                 Focus type = "FOCUS_INDUSTRY"
             ]
             accountinglabel = "FOCUS_INDUSTRY_LABEL"
-            priority = [[EARLY_PRIORITY]]
+            priority = [[DEFAULT_PRIORITY]]
```

Comment:
Looks like I messed up somewhere then and got, well, lucky in testing that it never processed in teh wrong order, they both need to apply before the multipliers (nothing ought to be prioritised at the same time as a multiplier)

I'll fix it tomorrow.